examples/browser/counter.js

```diff
@@ -124,12 +126,16 @@ function showInteractions(thing) {
                     eventSubscriptions[evnt] = true;
                     thing
                         .subscribeEvent(evnt, async function (data) {
-                            console.log('Event "' + evnt + '"');
-                            updateProperties();
```

Comment:
since you remove the `updateProperties()` call the event triggers the event information but does not update the `count` values on the left in the UI. Is this desired?

Reply:
I understand that technically it is not needed but in the UI the counter value is not updated. Hence the value in the UI shows the correct value if the event is fired while on the left the "same" counter value still shows the old value...

Reply:
I think the whole point of observing the `change` event is that I get notified if _someone else_ triggers a `count` change.
This can be that the UI is open in several browser windows or by Postman actions etc..

Reply:
Right, let's do it like this. I'll fix it in this PR, but I'll also open an issue to propose this change in the browser example: observe properties and change the event from the current logic to an even fired every time that the count is divisible by 2. I guess it would make a better newbie example.